common/src/main/resources/data/conditional/recipes/farm_and_charm/drying/dried_corn_from_ce.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "doapi:conditional",
+  "recipes": [
+    {
+      "conditions": [
+        {
+          "type": "doapi:not",
+          "value": {
+            "type": "forge:mod_loaded",
+            "modid": "brewery"
+          }
+        }
+      ],
+      "recipe": {
+        "type": "farm_and_charm:drying",
+        "ingredient": {
+          "item": "farm_and_charm:corn"
+        },
+        "recipe_type": "MEAT",
+        "result": {
+          "item": "cornexpansion:dried_corn",
```

Comment:
We can probably leave our dried corn even if brewery is loaded and use tags in recipes instead ?
A bit less clean, but easier to manage, I'll let you decide the way you prefer

Reply:
Edit: we should actually leave brewery's dried corn to avoid recipe conflicts (brewery doesn't have a dried_corn tag we can use), and use the dried_corn tag in our recipes instead, so both brewery's and our work, with brewery prevailing